source/codegen/templates/service_helpers.mako

```diff
@@ -158,13 +158,18 @@ ${initialize_standard_input_param(function_name, parameter)}\
   request_snippet = f'request->{field_name}()'
   c_type = parameter['type']
   c_type_pointer = c_type.replace('[]','*')
+  c_type_underlying_type = common_helpers.get_underlying_type_name(c_type)
 %>\
 % if c_type == 'ViConstString':
       ${c_type} ${parameter_name} = ${request_snippet}.c_str();\
 % elif c_type == 'ViString' or c_type == 'ViRsrc':
       ${c_type} ${parameter_name} = (${c_type})${request_snippet}.c_str();\
 % elif c_type == 'ViInt8[]' or c_type == 'ViChar[]':
       ${c_type_pointer} ${parameter_name} = (${c_type[:-2]}*)${request_snippet}.c_str();\
+% elif c_type == 'ViSession[]':
+      auto ${parameter_name}_request = ${request_snippet};
+      std::vector<${c_type_underlying_type}> ${parameter_name};
+      std::transform(${parameter_name}_request.begin(), ${parameter_name}_request.end(), std::back_inserter(${parameter_name}), [&](auto session_) {return session_repository_->access_session(session_.id(), session_.name()); }); \
```

Comment:
1. Add space before "return" statement.
2. Parameter to lambda should be `session` (w/o underscore). I believe in CPP, underscore is suffixed to private fields of the class.

Reply:
Done.